include/Surelog/SourceCompile/SymbolTable.h

```diff
@@ -50,16 +49,21 @@ class SymbolTable final {
   // Register given "symbol" string as a symbol and return its id.
```

Comment:
This comment needs updating: Returns a the SymbolId and the canonicalized string.